src/main/java/uk/ac/ebi/eva/pipeline/CustomJobLauncherCommandLineRunner.java

```diff
@@ -0,0 +1,36 @@
+package uk.ac.ebi.eva.pipeline;
+
+import org.springframework.batch.core.JobExecutionException;
+import org.springframework.batch.core.explore.JobExplorer;
+import org.springframework.batch.core.launch.JobLauncher;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.autoconfigure.batch.JobLauncherCommandLineRunner;
+import org.springframework.stereotype.Component;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.Properties;
+
+@Component
```

Comment:
Please add some documentation explaining the purpose of this class and which test it is associated with.